### PR TITLE
Eliminate redundant global stores in SM90 MQA logits kernels

### PR DESCRIPTION
**Pull request by Gregory-Pereira**
Seeks to address: #279

### CODE REVIEW

**Comment by llag9810** [comment]
Hi, I noticed this PR was closed. Could you share why? I am seeing a similar slowdown with fp8_mqa_logits on long-context prefill and want to understand the reason. Thanks.
